content/en/tracing/llm_observability/sdk.md

```diff
@@ -307,52 +422,135 @@ def process_message(prompt):
         tags={"host": "host_name"},
     )
     return resp
+
+@embedding(model_name="text-embedding-3", name="openai_embedding")
+def perform_embedding():
+    ... # user application logic
+    LLMObs.annotate(
+        span=None,
+        input_data={"text": "Hello world!"},
+        output_data=[0.0023064255, -0.009327292, ...],
+        metrics={"input_tokens": 4},
+        tags={"host": "host_name"},
+    )
+    return
+
+@retrieval(name="get_relevant_docs")
+def similarity_search():
+    ... # user application logic
+    LLMObs.annotate(
+        span=None,
+        input_data="Hello world!",
+        output_data=[{"text": "Hello world is ...", "name": "Hello, World! program", "id": "document_id", "score": 0.9893}],
+        tags={"host": "host_name"},
+    )
+    return
+
 {{< /code-block >}}
 
-## Persisting a span across contexts
+## Submitting evaluation metrics
 
-To manually start and stop a span across different contexts or scopes:
+To submit evaluation metrics for a span to Datadog:
 
-1. Start a span manually using the same methods (for example, the `LLMObs.workflow` method for a workflow span), but inline rather than as a context manager. 
-2. Pass the span object as an argument to other functions.
-3. Stop the span manually with the `span.finish()` method.
+1. Extract the span context from the given span by using `LLMObs.export_span(span)`. If `span` is not provided (as when using function decorators), the SDK will export the current active span.
+2. Use `LLMObs.submit_evaluation()` with the extracted span context and evaluation metric information.
```

Comment:
Should probably add a third step that shows where/how they can see the custom eval metric represented on the Datadog UI

Reply:
I think the problem is that it currently in flux -- this can be a TODO for me and we can decide what to do from the UI side (if anything, I also support evan's point that it should be self-explanatory) once we've settled on a final experience.